commons/commons-model-extended-mapping/src/main/kotlin/ebi/ac/uk/extended/mapping/to/ToSubmission.kt

```diff
@@ -18,11 +21,14 @@ fun ExtSubmission.toSimpleSubmission(): Submission = Submission(
 )
 
 private fun ExtSubmission.getSubmissionAttributes(): List<Attribute> {
-    val subAttrs = attributes.map { it.toAttribute() }.toMutableSet()
+    val subAttrs =
+        attributes
```

Comment:
I think we should create a private method `getpageTabAttributes`  to make clear why we remove the validator.